packages/nodes-base/nodes/GraphQL/GraphQL.node.ts

```diff
@@ -421,30 +424,29 @@ export class GraphQL implements INodeType {
 					requestOptions.qs.query = gqlQuery;
 				} else {
 					if (requestFormat === 'json') {
+						const jsonBody = requestOptions.body as IDataObject;
 						requestOptions.body = {
-							...requestOptions.body,
+							...jsonBody,
```

Comment:
we should definitely update the code to code like this, but I think this was the wrong PR to do this. This change broke the graphql node because on line 434 `jsonBody` isn't an object.